### PR TITLE
Add descriptions of each data source

### PR DESCRIPTION
**Pull request by orangejulius**
Consider this a first draft :)

Do we want to add pictures for some datasets coverage? In particular the coverage maps for quattroshapes and openaddresses are pretty nice, and might help break up the wall of text appearance of this page.

### CODE REVIEW

File: getting-started/006-data-sources.md [hunk 1]
```diff
@@ -1,8 +1,41 @@
...
+
+## Quattroshapes
+
+[Quattroshapes](http://quattroshapes.com/) provides geometry data for both administrative areas
+(countries, states, cities, and so on), and neighborhoods. The administrative area boundaries are derived from
+authoritative government data, while the neighborhoods are calculated from Foursquare check-ins, and
+geo-tagging data from Flickr. Because of the size of the datasets from Flickr and Foursquare,
+Quattroshapes provides a unique and global dataset for a difficult problem.
+
+Because of its global coverage and high quality, Mapzen Search uses Quattroshapes to determine the
```
Comment (riordan) [inline getting-started/006-data-sources.md]: Jumps right into `administrative areas`, which is a bit jargon-y.


Try this on for size:

Quattroshapes provides us with global coverage of location data for:
- countries
- regions (states/provinces)
- counties
- localities (cities, towns, hamlets, villages)
- and in many places, neighbourhoods.

Originally assembled by Foursquare, Quattroshapes provides not only the organizational hierarchy for nearly any point or address worldwide (town > local government > province > country), but also the borders for each of these places. Mapzen Search uses data from Quattroshapes to apply a consistent hierarchy to our data from other sources, so you can be sure that points of interest have consistent data about the cities, regions, and countries they're located in.

File: getting-started/006-data-sources.md [hunk 2]
```diff
@@ -1,8 +1,41 @@
...
+addresses around the world. Unlike OpenStreetmap, OpenAddresses data comes exclusively from regional
+authorities such as federal, state, or local governments. Because it consists of entirely bulk
+imports, OpenAddreses is a large, global, and rapidly growing dataset. Many countries, particularly
+in Europe, now have every address represented in OpenAddresses.
+
+## Geonames
+
+[Geonames](http://www.geonames.org/) is an aggregation of numerous authoritative and
+non-authoritative datasets. It contains information on everything from country borders to airport
+names to geographical features.
```
Comment (riordan) [inline getting-started/006-data-sources.md]: Geonames represents all places as a single point (not shapes encompassing their geography), and carries with it its own distinct hierarchy data that is not uniformly compatible with all data from other sources. Don't worry, we expect this should be resolved in the near future.

**Comment by rmglennon** [comment]
thanks @riordan. Working on this now. Any other comments?

**Comment by rmglennon** [comment]
@orangejulius @riordan One thing that might be good to do here (that @riordan's edits start to do) is relate this more to Mapzen Search. Some background on each data source is good (as some people may not have heard of all of these), but what info is Mapzen Search using from them? For example, what are some common items that we pull from OSM?

**Comment by riordan** [comment]
We should also add descriptions of the licensing regime for each data source as well as a general license descriptor.

**Comment by rmglennon** [comment]
Giving up on merging this PR. I copied the text into my own files and will add an issue to track the items in these comments.

**Pull request closed without merging**